Source/Diagnostics/ComputeDiagFunctors/PartPerGridFunctor.cpp

```diff
@@ -17,12 +17,17 @@ PartPerGridFunctor::operator()(amrex::MultiFab& mf_dst, const int dcomp) const
 {
     auto& warpx = WarpX::GetInstance();
     const Vector<long>& npart_in_grid = warpx.GetPartContainer().NumberOfParticlesInGrid(m_lev);
-    // MultiFab containing number of particles per grid
+    // Temporary MultiFab containing number of particles per grid.
     // (stored as constant for all cells in each grid)
+    const int ng = 1;
```

Comment:
same here

Reply:
added a comment in the code mentioning that for a cell-centered output multifab, the guard cell data of the source MF will not be accessed. To keep the code generic, we use ng=1, since for nodal output MF, we will need guard cell data for interpolation.